README.md

```diff
@@ -43,8 +45,15 @@ $ run_docker_ci_job # optional (--prune)
 
 **Evaluator** - evaluates something (parse tree, AST, or opcodes) directly; a "compiler" converts one thing into another
 
+**Register-based machines** - Set of virtual registers. The result is always in the "accumulator" register (i.e %eax)
```

Comment:
> The result is always in the "accumulator"

Not sure that's true.

Look at some LLVM bytecode. There is no distinct "accumulator" register.

(In fact, LLVM bytecode is SSA, so there could never be an accumulator register)

Reply:
removed https://github.com/softdevteam/pavel.yaiwr/pull/10/commits/42786f2650d13a08879ebaafc509037b744c1433

Reply:
OK, well you didn't have to remove the whole entry, but as you wish!